alerts/plugindefinition.yaml

```
@@ -62,6 +62,10 @@ spec:
       default: true
       required: false
       type: bool
+    - name: alerts.dashboard.create
+      description: Whether to install the default dashboards for self-monitoring Prometheus Alertmanger. 
+      default: false
```

don't we always want this?
```suggestion
      default: true
```

---

Good point. Let's opt out if necessary